### ISSUE TITLE
Hide toolbar until hover

### ISSUE THREAD

**Issue by rconstantine** [Ross-Patterson/Portals-Desktop-Organization#243]
Something I miss from Fences is the auto-hiding header for each portal. The current 'hide header' feature doesn't keep the layout the same. Presently, if you hide the header, all icons move up to fill the space left by the header. It appears the contents and the header both share a larger container. What would be better is if the header simply became invisible, but was still there. I think then the icons wouldn't all move up.

Does that make sense? Did I describe that OK?

**Comment by rconstantine** [comment]
This would literally be the opposite of the Auto Expand on Hover feature - instead of the header visible and the contents hidden until hover, the contents would be visible and the header hidden until hover.

**Comment by Ross-Patterson** [comment]
I understand, but then you would just be left with a big gap above/below your icons, why not just leave it visible in that case?

**Comment by rconstantine** [comment]
Yes, just fully transparent. So the container which I assume holds both header and contents would not change size, nor would icons need to move. Indeed, when I used to use Fences, I would snap the top of one fence against the bottom of another. Then, when the headers disappeared, they were nicely spaced apart.

**Comment by Ross-Patterson** [comment]
Got it, I would need to find a way to make it work with the existing hide header setting which correctly expands the content to fill the entire width/height.

**Comment by rconstantine** [comment]
Just made a donation. Note that I only did $0 initially because I couldn't actually test everything I wanted to that was important to me. You may want to do full-featured-time-limited instead of feature-limited trial version. Or turn on some of the disabled features by default, like the double-click to hide all, or allowing more than two portals. I don't recall what else I was annoyed with not being able to try. I just fear that many who would pay just end up doing 0 to try and then forget to donate later.

**Comment by rconstantine** [comment]
> Got it, I would need to find a way to make it work with the existing hide header setting which correctly expands the content to fill the entire width/height.

Yeah, having both features on at the same time would be bad. lol

**Comment by SourPatchKidd** [comment]
I second this notion. Having a "show header on hover" feature would be great! It might be visually jarring for those with larger headers, but for those of us with smaller headers, it's not a big deal if the icons are just moved down a bit. I suspect that those with larger headers probably just care about being able to see where things are, anyway.

**Comment by rconstantine** [comment]
@Ross-Patterson , perhaps you could just make everything in the header transparent instead of actually hide them? If the colors of everything in it allows for an alpha channel, that shouldn't be too hard. As for the old hide header mode, I would think the user would have to select either/or, and not both at once, right?
